save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/PluginType.kt

```diff
@@ -0,0 +1,22 @@
+package com.saveourtool.save.domain
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Enum of test type
+ * @property pluginName
+ */
+@Serializable
+@Suppress("CUSTOM_GETTERS_SETTERS")
+enum class PluginType(val pluginName: String) {
```

Comment:
Can we reuse stuff from save-core here somehow?

Reply:
Don't we use save-core in jvm part only?

Reply:
No, it's available for all platforms. It even has JS publication specifically to be used from save-frontend

Reply:
Well, maybe we will need to rename `TestConfigSections` in `save-cli` as well as add some readable name like `WarnPlugin` or something.

Reply:
Done

Reply:
We are using `plugin::class.simpleName!!` in `TestDiscoveringService.getAllTests`, `TestConfigSections` are only used in save-cli internally, IIRC

Reply:
Yes, we do, but we still need to tell one plugin from another, that's why I use this enum. To my mind we still need to add pluginName there